QHotkey/qhotkey_win.cpp

```diff
@@ -235,7 +235,7 @@ quint32 QHotkeyPrivateWin::nativeKeycode(Qt::Key keycode, bool &ok)
 
 	default:
 		if(keycode <= 0xFFFF)
-			return (byte)keycode;
+			return (BYTE)keycode;
```

Comment:
Makes sense. Could you also change to `static_cast`?
```suggestion
			return static_cast<BYTE>(keycode);
```

Reply:
Sounds good... Let me try that.